test/jdk/javax/swing/JComboBox/EditableComboBoxPopupPos.java

```diff
@@ -0,0 +1,215 @@
+                runTestOnComboBox(cb2Point, cb2Width, cb2Height);
+                runTestOnComboBox(cb3Point, cb3Width, cb3Height);
+                runTestOnComboBox(cb4Point, cb4Width, cb4Height);
+
+                SwingUtilities.invokeAndWait(() -> {
+                    cb1Str = cb1.getSelectedItem().toString();
+                    cb2Str = cb2.getSelectedItem().toString();
+                    cb3Str = cb3.getSelectedItem().toString();
+                    cb4Str = cb4.getSelectedItem().toString();
+                });
+
+                checkSelection(cb1Str, cb2Str, cb3Str, cb4Str);
+            } finally {
+                SwingUtilities.invokeAndWait(() -> frame.dispose());
+            }
+        }
+    }
+
+    private static void setLookAndFeel(UIManager.LookAndFeelInfo laf) {
+        try {
+            UIManager.setLookAndFeel(laf.getClassName());
+        } catch (UnsupportedLookAndFeelException ignored){
+            System.out.println("Unsupported LookAndFeel: " + laf.getClassName());
+        } catch (ClassNotFoundException | InstantiationException |
+                IllegalAccessException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    private static void runTestOnComboBox(Point p, int width, int height)
+            throws InterruptedException, InvocationTargetException {
+        if (lafName.equals("Mac OS X")) {
+            if (p == cb3Point || p == cb4Point) {
+                // Set isPopDown property to ensure popup matches native MacOS
+                SwingUtilities.invokeAndWait(() -> {
+                    cb3.putClientProperty("JComboBox.isPopDown", Boolean.FALSE);
+                    cb4.putClientProperty("JComboBox.isPopDown", Boolean.FALSE);
+                });
+
+                robot.mouseMove(p.x + width - BUTTON_OFFSET,
+                        p.y + (height / 2) + POPUP_OFFSET);
+                robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
+                robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
+
+                robot.mouseMove(p.x + (width / 2) - BUTTON_OFFSET,
+                        p.y + POPUP_OFFSET);
+                robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
+                robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
+            } else {
+                robot.mouseMove(p.x + width - BUTTON_OFFSET,
+                        p.y + (height / 2) + POPUP_OFFSET);
+                robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
+                robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
+
+                robot.mouseMove(p.x + (width / 2) - BUTTON_OFFSET,
+                        p.y + height + POPUP_OFFSET - 8);
+                robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
+                robot.mouseRelease(InputEvent.BUTTON1_DOWN_MASK);
+            }
+        } else {
+            robot.mouseMove(p.x + width - BUTTON_OFFSET,
+                    p.y + (height / 2) + POPUP_OFFSET);
```

Comment:
I am having a solid 10 sec delay with Motif LaF for forth ComboBox.
We are moving mouse really close to the bottom right corner, so resize mouse cursor appears `⤡`.
This somehow triggers [JDK-8299937](https://bugs.openjdk.org/browse/JDK-8299937)  behavior.

Without adding `POPUP_OFFSET` all works fine(tested on Mac and Ubuntu).